src/main/java/Duke.java

```diff
@@ -1,10 +1,56 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final Tasks taskManager = new Tasks();
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greetUser();
+        echo();
+        exit();
+    }
+    private static void greetUser() {
+        System.out.println("________________________________________");
+        System.out.println(" Hello! I'm Judy");
+        System.out.println(" What can I do for you?");
+        System.out.println("________________________________________");
+    }
+
+    private static void echo() {
+        Scanner scanner = new Scanner(System.in);
+        String command;
+
+        do {
+            try {
+                command = scanner.nextLine();
+                System.out.println("____________________________________");
+                if(command.equalsIgnoreCase("bye")) {
+                    break;
+                } else if (command.equals("list")) {
+                    taskManager.listOutTasks();
+                } else if (command.startsWith("mark")){
+                    taskManager.markTask(command);
+                } else if (command.startsWith("unmark")) {
+                    taskManager.unmarkTask(command);
+                } else if (command.startsWith("delete")) {
+                    taskManager.deleteTask(command);
+                } else {
+                    taskManager.addTask(command);
+                }
+            } catch (DukeException e) {
+                System.out.println(e.getMessage());
+            }
```

Comment:
This can written as a _switch case_  for better readability, example:

```java
String input = scanner.nextLine().split(" ");
command = input.split(" ")[0];
switch (command.toLowerCase()) {
case "bye":
   ...
case ...:
   ...
default:
   ...
}
```

see:  
https://nus-cs2103-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-deep-nesting